components/gc-most-requested/index.json-ld

```diff
@@ -85,6 +85,10 @@
 				"_:implement_mostrequested"
 			],
 			"history": [
+				{
+					"en": "November 2024 - Items are now distributed vertically instead of horizontally.",
+					"fr": "Novembre 2024 - Les items sont maintenant distribués verticalement plutôt que horizontalement."
```

Comment:
Let's update that date for like May 2025 which is closer to the reality, or you can add an additional item and prepend "Draft" to this item